services/core-web/src/components/mine/ExplosivesPermit/MineExplosivesPermitTable.tsx

```diff
@@ -266,209 +489,157 @@ const MineExplosivesPermitTable: FC<RouteComponentProps & MineExplosivesPermitTa
-                  cancelText="Cancel"
-                >
-                  <Button ghost type="primary" size="small">
-                    <img src={TRASHCAN} alt="Remove Permit" />
-                  </Button>
-                </Popconfirm>
-              </AuthorizationWrapper>
+          <Row justify="space-between">
+            <Col>
+              <Typography.Text>{record.permit_number || Strings.EMPTY_FIELD}</Typography.Text>
+            </Col>
+            {record.explosives_permit_amendments.length > 1 && (
+              <Col className="amendments-badge">
+                <FontAwesomeIcon icon={faFiles} />
+                {record.explosives_permit_amendments.length - 1}
+              </Col>
             )}
-          </div>
+          </Row>
         );
       },
     },
+    renderTextColumn("mines_permit_number", "Mines Act Permit #", false, Strings.EMPTY_FIELD),
+    renderTextColumn("now_number", "Notice of Work #", false, Strings.EMPTY_FIELD),
+    {
+      title: "Amendments",
+      key: "explosives_permit_amendments",
+      render: (record) => {
+        return (
+          <Typography.Text>{record?.explosives_permit_amendments.length - 1 || 0}</Typography.Text>
+        );
+      },
+    },
+    {
+      title: "Status",
+      key: "is_closed",
+      render: (record) => {
+        return <Typography.Text>{record.is_closed ? "Closed" : "Open"}</Typography.Text>;
+      },
+    },
+    renderDateColumn("expiry_date", "Expiry Date", false, null, Strings.EMPTY_FIELD),
+    {
+      title: (
+        <Row>
+          <Typography.Text className="margin-medium--right">Explosives</Typography.Text>
+          <CoreTooltip
+            title="This is the total quantity stored on site. Click to view more details"
+            icon="question"
+            iconColor={COLOR.darkGrey}
+          />
+        </Row>
+      ),
+      key: "total_explosive_quantity",
+      dataIndex: "total_explosive_quantity",
+      render: (text, record) => (
+        <div
+          className="underline"
+          onClick={(event) => handleOpenViewMagazineModal(event, record, "DET")}
+        >
+          {text || "0"} kg
+        </div>
+      ),
+    },
+    {
+      title: (
+        <Row>
+          <Typography.Text className="margin-medium--right">Detonators</Typography.Text>
+          <CoreTooltip
+            title="This is the total quantity stored on site. Click to view more details"
+            icon="question"
+            iconColor={COLOR.darkGrey}
+          />
+        </Row>
+      ),
+      key: "total_detonator_quantity",
+      dataIndex: "total_detonator_quantity",
+      render: (text, record) => (
+        <div
+          className="underline"
+          onClick={(event) => handleOpenViewMagazineModal(event, record, "DET")}
+        >
+          {text || "0"} units
+        </div>
+      ),
+    },
+    actionsColumn,
+  ];
+
+  const amendmentDetailColumns: ColumnType<amendmentsWithTotal>[] = [
```

Comment:
Are the status, expiry date, total explosive quantity, total detonator quantity, and actions column the same between both sets of columns? I'd be tempted to define it as esupCommonColumns or something and [...spread] it in. Not necessary though.

Reply:
Hmmm, good thought.  Yeah I they're the same.  Updated!